man/udev.xml

```diff
@@ -115,7 +116,23 @@
           </listitem>
         </varlistentry>
       </variablelist>
+      </refsect2>
+
+      <refsect2><title>Values</title>
+        <para>Values are written as double quoted strings, such as <literal>"string"</literal>.
+        To include a <literal>"</literal> character in the value, use <literal>\"</literal>.
```

Comment:
`<literal>` looks bad here. Is there a better tag for this?

udev.html
![image](https://user-images.githubusercontent.com/4318379/97579945-a6401480-1a2d-11eb-87a8-899703cdc813.png)
Even worse in udev.7
![image](https://user-images.githubusercontent.com/4318379/97580053-ccfe4b00-1a2d-11eb-806d-6c9b4055a527.png)

Reply:
We use `(")` in such cases sometimes. So maybe
```
To include a quation mark (") in the value, precede it by a backslash (\").
```